README.md

```diff
@@ -151,4 +151,4 @@ Only update the `Unreleased` section. Please use the template below when updatin
 * r/tfe_resource: description of change or bug fix ([#124](link-to-PR))
 ```
 
-Change categories: `BUG FIXES`, `ENHANCEMENTS`, `FEATURES`, `NOTES`
+Change categories: `BUG FIXES`, `ENHANCEMENTS`, `FEATURES`, `NOTES`, `BREAKING_CHANGE`
```

Comment:
Small thing, but could this be consistent with the others as `BREAKING CHANGES`?
```suggestion
Change categories: `BUG FIXES`, `ENHANCEMENTS`, `FEATURES`, `NOTES`, `BREAKING CHANGES`
```

You could potentially include the same list that's in the [go-tfe README](https://github.com/hashicorp/go-tfe/blob/master/README.md) if you think it would be helpful. I believe I originally pulled these from [the docs on provider versioning/changelogs](https://www.terraform.io/docs/extend/best-practices/versioning.html).

```
BREAKING CHANGES: Use this for any changes that aren't backwards compatible. Include details on how to handle these changes.
FEATURES: Use this for any large new features added,
ENHANCEMENTS: Use this for smaller new features added
BUG FIXES: Use this for any bugs that were fixed.
NOTES: Use this section if you need to include any additional notes on things like upgrading, upcoming deprecations, or any other information you might want to highlight.
```